src/Services/Air/Air.js

```diff
@@ -135,33 +135,31 @@ module.exports = (settings) => {
     },
 
     ticket(options) {
-      return retry({ retries: 3 }, (again, number) => {
-        if (settings.debug && number > 1) {
-          console.log(`ticket ${options.pnr} retry number ${number}`);
-        }
-        return (options.ReservationLocator
-            ? Promise.resolve(options)
-            : this.getPNR(options).then((booking) => {
-              const ticketParams = Object.assign({}, options, {
-                ReservationLocator: booking.uapi_reservation_locator,
-              });
-              return ticketParams;
-            })
-        ).then(
-          ticketParams => service.ticket(ticketParams)
+      return (options.ReservationLocator
+        ? Promise.resolve(options.ReservationLocator)
```

Comment:
@articice I have coverage issue here.
Could the `ReservationLocator` be directly passed on the first call?